src/lib.rs

```diff
@@ -824,454 +810,3 @@ impl Default for Section {
         Section::new()
     }
 }
-
```

Comment:
removing these tests wasn't intentional i just did not want to deal with the merge conflicts from em